Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Private/InteropCodeGenerator/TypeBindingGenerator.h

```diff
@@ -40,7 +40,8 @@ void GenerateTypeBindingHeader(
 	FString SchemaFilename,
 	FString InteropFilename,
 	UClass* Class,
-	const TSharedPtr<FUnrealType> TypeInfo);
+	const TSharedPtr<FUnrealType> TypeInfo,
+	TMap<FString, FString>& GeneratedStructInfo);
```

Comment:
Nit: typedef `TMap<FString, FString>` to something more descriptive of what's in there + add a comment.

Reply:
Done